app/actions/views/post.js

```diff
@@ -292,42 +351,39 @@ function getPostsAdditionalDataBatch(posts = []) {
             // do nothing
         }
 
-        return actions;
+        return {data};
     };
 }
 
-function profilesStatusesAndToLoadFromPosts(posts = []) {
-    return (dispatch, getState) => {
-        const state = getState();
-        const {currentUserId, profiles, statuses} = state.entities.users;
+function profilesStatusesAndToLoadFromPosts(state, posts = []) {
```

Comment:
This name has aways confused me. Perhaps, `userMetadataToLoadFromPosts`? I'm 0/5 on the proposed alternative, and 1/5 on renaming.

With consolidation/batching of functions we naturally are stuck with long function names with a lot of conjunctions (`and` this `and` that) in the name. I tend to prefer more explicit naming, even if the names get really long.